packages/gafl-webapp-service/src/processors/__tests__/date-and-time-display.spec.js

```diff
@@ -13,28 +13,36 @@ describe('displayStartTime', () => {
     const startTime = displayStartTime({ licenceStartDate: '2021-01-01', licenceStartTime: '12' })
     expect(startTime).toEqual('Friday, January 1st, 2021, 12:00pm (midday)')
   })
-  it('displays the date in the correct format where the start time is 3pm', () => {
+  it('displays the date in the correct format where the start time is 3pm (UTC)', () => {
     const startTime = displayStartTime({ licenceStartDate: '2021-01-01', licenceStartTime: '15' })
     expect(startTime).toEqual('Friday, January 1st, 2021, 3:00pm')
   })
+  it('displays the date in the correct format where the start time is 3pm (BST)', () => {
+    const startTime = displayStartTime({ licenceStartDate: '2021-07-01', licenceStartTime: '15' })
+    expect(startTime).toEqual('Thursday, July 1st, 2021, 3:00pm')
+  })
 
   it('displays the date in the correct format where the API start time is midnight', () => {
     // Tests that the API start time is used if the date/time has just rolled over (purchase completed at 11.59.59pm)
     const startTime = displayStartTime({ startDate: '2021-01-01T00:00:00.000Z', licenceStartDate: '2020-12-31' })
     expect(startTime).toEqual('Friday, January 1st, 2021, 12:00am (midnight)')
   })
   it('displays the date in the correct where the API start time is 1 minute past midnight', () => {
-    const endTime = displayStartTime({ startDate: '2020-01-01T00:01:00.000Z', licenceStartDate: '2020-01-01' })
-    expect(endTime).toEqual('Wednesday, January 1st, 2020, 12:01am')
+    const startTime = displayStartTime({ startDate: '2020-01-01T00:01:00.000Z', licenceStartDate: '2020-01-01' })
+    expect(startTime).toEqual('Wednesday, January 1st, 2020, 12:01am')
   })
   it('displays the date in the correct format where the API start time is 12pm', () => {
     const startTime = displayStartTime({ startDate: '2021-01-01T12:00:00.000Z', licenceStartDate: '2020-01-01', licenceStartTime: '12' })
     expect(startTime).toEqual('Friday, January 1st, 2021, 12:00pm (midday)')
   })
-  it('displays the date in the correct format where the API start time is 3pm', () => {
+  it('displays the date in the correct format where the API start time is 3pm (UTC)', () => {
     const startTime = displayStartTime({ startDate: '2021-01-01T15:00:00.000Z', licenceStartDate: '2021-01-01', licenceStartTime: '15' })
     expect(startTime).toEqual('Friday, January 1st, 2021, 3:00pm')
   })
+  it('displays the date in the correct format where the API start time is 3pm (BST)', () => {
+    const startTime = displayStartTime({ startDate: '2021-07-01T14:00:00.000Z', licenceStartDate: '2021-07-01', licenceStartTime: '15' })
```

Comment:
If this is a BST date string, shouldn't it be `2021-07-01T14:00:00.000+01:00`?

Reply:
The Sales API and Dynamics always use UTC to represent dates.  Therefore they will always return `2021-07-01T14:00:00.000Z` rather than `2021-07-01T15:00:00.000+01:00`